wsgi.py

```diff
@@ -0,0 +1,4 @@
+from app import app
```

Comment:
Really not much to say. It is perfect, and does exactly what the challenge asks of you. Code is very clean and easy to read.  Neat design and added favicon. Amazing job :D

Overall 4/10